networkx/algorithms/tests/test_cluster.py

```diff
@@ -170,13 +176,15 @@ def test_triangle_and_edge(self):
         G = nx.cycle_graph(3, create_using=nx.DiGraph())
         G.add_edge(0, 4, weight=2)
         assert nx.clustering(G)[0] == 1.0 / 6.0
-        assert nx.clustering(G, weight="weight")[0] == 1.0 / 12.0
+        np.testing.assert_allclose(nx.clustering(G, weight="weight")[0], 1.0 / 12.0)
```

Comment:
It may be worth adding a comment above these tests to indicate why we aren't doing exact comparisons.  Maybe something like:
```
# Relaxed comparisons to allow graphblas-algorithms to pass tests
```

Reply:
Done.